app/packages/core/src/plugins/SchemaIO/components/index.ts

```diff
@@ -21,7 +21,6 @@ export { default as HelpTooltip } from "./HelpTooltip";
 export { default as HiddenView } from "./HiddenView";
 export { default as ImageView } from "./ImageView";
 export { default as InferredView } from "./InferredView";
-export { default as JSONView } from "./JSONView";
```

Comment:
Sorry for the late comment, but, although this may be rarely used in operators, we still have the support for `JSONView` in operators IO and will likely need to add this back.

Reply:
My mistake. Thanks!

Reply:
It's all good. These components do seem like they are unused as they are dynamically referenced 😄

I can re-use the JSONView implementation from panel and bring it over here in the afternoon.